cfep-01.md

```diff
@@ -0,0 +1,135 @@
+organization operates.  These change can be technical or address the
+social aspects of the organization.  Small changes often do not need a CFEP and
+can be discussed on the
+[conda-forge gitter channel](https://gitter.im/conda-forge/conda-forge.github.io),
+the [conda-forge mailing list](https://groups.google.com/forum/#!forum/conda-forge),
+or as part of a conda-forge meeting.  More involved or controversial changes
+should be submitted as CFEPs.  When it is unclear if a change requires a CFEP,
+ask in one of the above forums and the conda-forge core team should be able to
+provide guidance.  The conda-forge core team may solicit CFEPs on topics of
+concern to the community.  CFEPs should focus on a single issue, broad changes
+should be split into multiple well-focused CFEPs.
+
+Each CFEP must have a champion or champions -- someone who will write the
+CFEP in the format described below, submit the CFEP as a pull request,
+follow the discussion on the pull request, and answer questions.  Before
+beginning to write a proposal, it is best to ascertain if the idea is CFEP-able.
+A discussion on the
+[conda-forge gitter channel](https://gitter.im/conda-forge/conda-forge.github.io),
+the [conda-forge mailing list](https://groups.google.com/forum/#!forum/conda-forge),
+or as part of a conda-forge meeting is the best way to go about this.
+
+Once the champion has asked the conda-forge community as to whether an idea
+has any chance of acceptance, a draft CFEP should be written following the
+template in [CFEP-00](cfep-00.md) and described below.
+
+The CFEP draft may be circulated to interested parties for informal comment and
+review prior to a formal submission. During this time, the Status of the CFEP
+should be set to Draft.
+
+## Submission
+
+Once complete, this draft should be submitted as a pull request to the
+[conda-forge/conda-forge-enhancement-proposals](https://github.com/conda-forge/conda-forge-enhancement-proposals)
+repository with the Status changed to Proposed.  At this point, members of the
+conda-forge community will review the submission.
+
+## Review
+
+CFEP review will begin once a pull request has been made.  All members of the
+conda-forge community are welcome and encouraged to participate in the review
+of CFEPs in a civil and respectful manner.  The CFEP champion(s) should be
+prepared to answer questions on the proposal and make updates to the
+proposal as recommended by the community.
+
+## Resolution
+
+All CFEPs will be resolved as either *Rejected*, *Accepted* or *Deferred*
+depending on the consensus of the community. Once the community has reached a
+consensus, the CFEP champion should update the Status of proposal and add a
+link to the discussion to the table.  Regardless of the resolution, the pull
+request should be merged once these tasks have been done.
+
+## CFEP Maintenance
+
+For *Rejected* CFEPs, no modifications are made after the pull request is merged.
```

Comment:
This may seem like a small point, but actually I can see value in doing this as a second step. Namely we merge in *Draft* mode and then later someone (not even necessarily the original author) updates this to rejected. The value being twofold.

1. We are not relying on someone that just had their CFEP rejected to do this work. This seems minor, but I can certainly see people leaving abandoned PRs that were rejected.
2. We can offer up a summary judgement. Why did we reject it. What changes might allow for a future proposal to be accepted. Etc.